### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by basirking**
** in this version we:**
** -Add media queries to make the website responsive**
** -Add Desktop version portfolio**
** -Add new features to the file**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -280,3 +289,169 @@ input[type=text],
...
+  .text-img {
+    font-weight: 200;
+    color: white;
+  }
+
+  .forms {
+    display: flex;
+    flex-direction: column;
+    width: 31%;
+  }
```
Comment (zieeco) [inline style.css:452]: - [ ] Please, kindly add the background image to your contact section.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @basirking 


Great work on your portfolio 👏 
You've done well implementing the design, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A 

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -280,3 +289,169 @@ input[type=text],
...
+    width: 1156px;
+    margin-left: 70px;
+    top: 0;
+    border-radius: 24px;
+    padding-top: 50px;
+    justify-content: center;
+    display: flex;
+    flex-direction: row;
+    align-content: center;
+  }
```
[the remark this answers is not in the document]
Comment (basirking) [reply style.css:384]: what exactly should be the width??

File: index.html [hunk 3]
```diff
@@ -164,45 +113,112 @@ <h5>Back End Dev</h5>
...
-            </ul>
+              </div>
+  
+              <div class="button">
+                <a href="https://github.com/basirking/">
+                  <h4>See Project</h4>
+                </a>
+              </div>
+            </div> 
+          </section>
```
Comment (keroloussamy) [inline index.html:146]: - [ ] Kindly as asked by the previous reviewer, change the color for the section to white, and around it make it this degree of gray.
- [ ] Kindy as asked by the previous reviewer, make the button like the design the background white and the border blue.

Your design
image

Figma design
image

File: index.html [hunk 4]
```diff
@@ -301,82 +317,97 @@ <h5>2015</h5>
...
                         <img src="imgs/Linkedinicon.svg" alt="width">
                     </a>  
                 </li>
                 <li>
-                    <a href="#">
+                    <a href="https://github.com/basirking/">
                         <img src="imgs/twitter.png" alt="width">
                     </a>  
                </li>
                <li>
```
Comment (keroloussamy) [inline index.html:378]: - [ ] Kindly as asked by the previous reviewer to follow the original design
- [ ] Kindly Make the button like the design the background white and the border blue.
- [ ] Kindly make the padding around the icons and the text equal to each other as the Figma design, and make the text on the left.

Your design
image

Figma design
image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @basirking, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

 ## Required Changes ♻️
_Check the comments under the review._





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aimemalaika — APPROVED** [review]
Hi @basirking, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**